lib/libutee/tee_api_operations.c

```diff
@@ -662,14 +662,24 @@ TEE_Result TEE_SetOperationKey(TEE_OperationHandle operation,
 	return res;
 }
 
+TEE_Result TEE_SetOperationKey(TEE_OperationHandle operation,
+			       TEE_ObjectHandle key)
+{
+	if (operation != TEE_HANDLE_NULL &&
```

Comment:
> Panic Reasons
> - If operation is not a valid opened operation handle
> - If the flag TEE_HANDLE_FLAG_INITIALIZED is set on the operation

An invalid operation should also cause a panic, or am I missing something?

Reply:
Correct, I'll fix